components/crossplane-config/staging/provider-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
```

Comment:
We need to add the ArgoCD annotation to allow skipping when the CRD hasn't been registered yet.